community/dbms/src/main/java/org/neo4j/commandline/dbms/MemoryRecommendationsCommand.java

```diff
@@ -199,6 +219,92 @@ public void execute( String[] args ) throws IncorrectUsage
         print( initialHeapSize.name() + "=" + heap );
         print( maxHeapSize.name() + "=" + heap );
         print( pagecache_memory.name() + "=" + pagecache );
+
+        boolean db = arguments.getBoolean( ARG_DB );
+        if ( !db )
+        {
+            return;
+        }
+        File configFile = configDir.resolve( Config.DEFAULT_CONFIG_FILE_NAME ).toFile();
+        File storeDir = getConfig( configFile ).get( database_path );
+
+        print( "#" );
+        print( "# A minimal page cache memory setting to fully map the database contents of '" + storeDir.getAbsolutePath() + "':" );
+        print( pagecache_memory.name() + "=" + dbSpecificPageCacheMemory( storeDir ) );
```

Comment:
Since this output is kind of meant to the copy-pasted more-or-less directly into `neo4j.conf`, I think we should only output one active `dbms.memory.pagecache.size`. So what I think should happen when `--database` is enabled, is that the `print( pagecache_memory.name() + "=" + pagecache );` line above should actually print the setting as commented out, like `print( "# " + pagecache_memory.name() + "=" + pagecache );`.